scripts/cronos-experimental-devnet.yaml

```diff
@@ -31,6 +31,7 @@ cronos_777-1:
       block:
         max_bytes: "1048576"
         max_gas: "81500000"
+        time_iota_ms: "2000"
```

Comment:
increase block time to reduce the likehood that tx is immediately included in batch for integration tests and making the cancellation impossible

Reply:
```suggestion
        # increase block time to reduce the likehood that tx is immediately included in batch for integration tests and making the cancellation impossible
        time_iota_ms: "2000"
```